complaint_search/tests/test_es_interface.py

```diff
@@ -303,6 +320,34 @@ def test_search_agg_exclude__valid(self, mock_rget, mock_scroll, mock_count, moc
     #     mock_search.assert_not_called()
     #     self.assertEqual(1, mock_rget.call_count)
 
+    @mock.patch.object(ElasticSearchExporter, 'export_csv')
+    @mock.patch('elasticsearch.helpers.scan')
+    def test_search_with_format_csv__valid(self, mock_search, mock_exporter):
```

Comment:
It took me a minute to figure out what this test is trying to test. There's a bit of complexity here, as understanding this test requires understanding the internals of the `search()` method being tested. The reader has to know that `search()` calls `elasticsearch.helpers.scan` and `ElasticSearchExporter.export_csv` internally.

What is the purpose of the `body` variable? Is it used anywhere?